devices_test.go

```diff
@@ -114,18 +150,15 @@ func TestAccDeviceBasic(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer deleteDevice(t, c, d.ID)
+	defer deleteDevice(t, c, d.ID, false)
 
 	if len(d.ProvisionEvents) != 10 {
```

Comment:
This is unrelated, but please remove the ProvisionEvents len check, it makes this test fail now (14.09.2020), as there's 11 not 10 prov events.

Reply:
You can do this by rebasing on current master.